lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-transformer/resources/transformation/ado2xwiki.atl

```diff
@@ -228,8 +285,43 @@ rule RootWebHome {
 	msc_p2:XWIKI!Property (
 		name <- 'name',
 		type <- 'String',
-		value <- 'Set of models' --TODO: there is no model set name defined in our process
+		value <- 'Set of models'
+	),
+	--begin documentation
+	msc1:XWIKI!Object (
```

Comment:
This object is not needed. There is already an instance of `LPCode.ModelSetClass` called `msc`.